experiment/simulation/css/main.css

```diff
@@ -1 +1,363 @@
-/* You CSS goes in here */
\ No newline at end of file
+.window { z-index:20; }
+.jtk-connector { z-index:4; }
+.jtk-endpoint { z-index:5; }
+.jtk-overlay { z-index:6; }
+
+.columns {
+ display: flex;
+}
+
+.drag-drop {
+  display: inline-block;
+  position: absolute;
+  top: 0px;
+  width: 50px;
+  height: 50px;
+  border-radius: 0.75em;
+  border: 0.2rem solid rgb(98, 178, 235);
+  transform: translate(0px, 0px);
+  z-index: 1;
+  background-size: 100%;
+}
+
+.logic-gate{
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.AND {

```

Comment:
Where is this class used? There's a class named as and used in index.html. If they are intended to be the same classes, please name this selector in small-case

Reply:
Yes, made all classes lowercase